platform/test/resources/TestUtility.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Compares two configuration objects. 
+ * @param {Object} configA
+ * @param {Object} configB
+ * @returns {Boolean} true if the objects values are equal.
+ */
+export function configObjectEquals(configA, configB){
+
+    const strConfigA = JSON.stringify(configA);
+    const strConfigB = JSON.stringify(configB);
+
+    return (strConfigA === strConfigB)
+}
+
+/**
+ * Prints a configuration object including its values.
```

Comment:
The function doesn't strictly "print" the object, it just returns its stringification. Given that that's the case, is the function actually needed or could it just be replaced with direct uses of `JSON.stringify`?

Reply:
Yes it's not particularly useful as it is so will use stringify directly.

Reply:
Removed in 52769f3d17c78207d5778a9ec46d8e090149bfb0